src/index.html

```diff
@@ -5,9 +5,48 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+      >
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card card--width" data-qa="card">
+      <img
+        class="card__photo"
+        src="./images/imac.jpeg"
+        alt="Imac Monoblock photo"
+      >
+      <h1 class="card__header">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+      <p class="card__code">Product code: 195434</p>
+
```

Comment:
1. Fix everywhere
<img width="1071" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/49764881/35a7c7cc-6cb4-4176-ab55-ef703bb907d6">
2. You can have only 1 `h1` per page, but we can have a lot of cards. I think `h3` fits better here.